### DISCUSSION TITLE
Promptfoo integration

### DISCUSSION THREAD

**Discussion by commit4ever** [langfuse/langfuse#3375] [Ideas]
### Describe the feature or potential improvement

I heavily use langfuse tracing and would like to manage all aspects of evals there as well or possibly an integration with https://github.com/promptfoo/promptfoo . 

Thanks!

**Comment by marcklingen** [comment]
Hi @aiseei, did you try this integration already? https://www.promptfoo.dev/docs/integrations/langfuse/

**Comment by AmineDjeghri** [comment]
It could be great if we can send the metrics score and other stuff from promptfoo to langfuse.
The actual promptfoo integration only manages to get the prompt from langfuse
